config/base/params-vllm-gaudi.env

```diff
@@ -0,0 +1 @@
+vllm-gaudi-image=quay.io/opendatahub/vllm:fast-gaudi
```

Comment:
I don't see this image in the quay.io. is it expected?

Reply:
thanks for pointed it out. We had recently change the gaudi branch in vLLM from gaudi-1.19 to main-gaudi. I miss to fix it openshift-ci pipeline. Image with `fast-gaudi` would be created once the pipeline would be fixed.

Reply:
@Jooho The Jira mentioned in the main description states 2.16 is the target.

Reply:
> Image with fast-gaudi would be created once the pipeline would be fixed.
So, I wonder... is this PR safe to merge, despite there is no image yet?

Reply:
I am working on its pipeline. Its image should be available by today evening

Reply:
I'm good with the PR @vaibhavjainwiz .
Let us know when it is safe to merge.

Reply:
I had fixed the openshift-ci pipeline for it and raised below PR to retrigger build. Image with tag `fast-gaudi` should be available soon. I would keep you updated.

Reply:
yes I see the image now.